READMEs/developers.md

```diff
@@ -36,7 +36,8 @@ cd ocean.py
 sudo apt-get install -y python3-dev gcc python-pytest
 
 #Initialize virtual environment and activate it.
-python -m venv venv
+#Use python3 to create the env for Python3.8, Python3.9, Pyhton3.10, Python3.11.
```

Comment:
there is a typo in Python3.10, also I think you can rephrase as: "make sure your Python version inside the venv is >=3.8."

Can you please add spaces after the `#` character throughout this file? It always bugs me and I don't want to perpetuate this